File: log/src/main/java/no/nav/familie/log/IdUtils.java
package no.nav.familie.log;

import java.util.UUID;

public final class IdUtils {
    private IdUtils() {
    }

    public static String generateId() {
        UUID uuid = UUID.randomUUID();
        return Long.toHexString(uuid.getMostSignificantBits()) + Long.toHexString(uuid.getLeastSignificantBits());
    }
}

File: log/src/main/java/no/nav/familie/log/IdUtils.kt
package no.nav.familie.log

import java.util.*

object IdUtils {
    @JvmStatic fun generateId(): String {
        val uuid = UUID.randomUUID()
        return java.lang.Long.toHexString(uuid.mostSignificantBits) + java.lang.Long.toHexString(uuid.leastSignificantBits)
    }
}
